spec/policies/user_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe UserPolicy do
+  let(:admin_user) { build(:user, admin: true) }
+  let(:other_user) { build(:user) }
+  let(:owner)      { build(:user) }
+  let(:user)       { build(:user) }
+  let(:app)        { build(:service_provider, user: owner) }
+
+  permissions :create? do
+    it 'allows any user to create' do
+      expect(ServiceProviderPolicy).to permit(user, app)
+    end
+  end
+
+  permissions :edit? do
+    it 'allows owner or admin to edit' do
```

Comment:
I realize the other policy spec is structured this way, but let's keep one `expect` per `it`, e.g.,
```
  it 'allows owner to edit' do...
  it 'allows admin to edit' do...
  it 'does not allow other users to edit' do...
```